servicemanager/servicemanager.go

```diff
@@ -51,6 +51,9 @@ type ServiceInfoProvider interface {
 	GetService(serviceID string) (ServiceInfo, error)
 	GetAllServices() ([]ServiceInfo, error)
 	AddService(ServiceInfo) error
+	GetAllServiceVersions(string) ([]ServiceInfo, error)
+	RemoveService(ServiceInfo) error
+	ActivateService(ServiceInfo) error
```

Comment:
Consider to have one method: `UpdateService`.

Reply:
decided to add update Service incase it will be necessary